### DISCUSSION TITLE
I need help uninstalling Udisk from steam deck

### DISCUSSION THREAD

**Discussion by FrontoOptimo** [storaged-project/udisks#1277] [Q&A]
why wont anyone answer my question on uninstalling. u guys close my topic  when not providing any information on uninstalling

**Comment by vojtechtrefny** [comment]
This is the fourth time you asked this question, you are not getting a different answer, the only answer we can give you is to ask your distribution vendor. As upstream we don't know how the package management is done in your distribution so we can't give you a better answer.

https://github.com/storaged-project/udisks/discussions/1141
https://github.com/storaged-project/udisks/discussions/1220
https://github.com/storaged-project/udisks/discussions/1276
